cleanlab_studio/internal/util.py

```diff
@@ -63,3 +194,11 @@ def check_none(x: Any) -> bool:
 
 def check_not_none(x: Any) -> bool:
     return not check_none(x)
+
+
+def quote(s: str) -> str:
+    return f'"{s}"'
+
+
+def quote_list(l: list) -> list:
+    return list(map(quote, l))
```

Comment:
just a style thing, but we typically prefer using list comprehensions over `list(map(...))` since it's a little more readable